frontend/src/pages/CourseSelector/courseTabs/CourseTabs.jsx

```diff
@@ -53,6 +67,12 @@ export const CourseTabs = () => {
           )}
         </Droppable>
       </DragDropContext>
+      {
+        !!tabs.length &&
+        <Dropdown overlay={TabDropdownMenu}>
+          <Button className="cs-tabs-dropdown-btn" icon={<EllipsisOutlined />}/>
```

Comment:
not sure if this icon is the best option though. possibly we just have it be a single click delete?

Reply:
Hmm yea.. There's not much UI convention for an 'Close all tabs' button (not that I know of). On chrome, you can close tabs either with a shortcut or a right click on the tab. I thought having it like an ellipsis icon would give a similar meaning to a right click.

If we want to go with a single click delete approach, I feel that a very powerful action to mass delete tabs if the user did not intend for that to happen.

I guess another approach is to change the icon to a single click delete but instead a warning popup appears something similar to this?
![image](https://user-images.githubusercontent.com/65492346/163673996-f53c1f42-8023-455a-b355-dbd3f547f3f2.png)

I prefer either using the elipsis and having a dropdown to close all tabs or changing the icon and having a popup for a warning. Thoughts?

Reply:
I think it depends on if we want to add more functionality under those ellipses.

If we do, then keeping it this way is reasonable, but if not, then I think changing the icon makes the most sense